README.md

```diff
@@ -156,6 +156,21 @@ To contribute, please check out [the contribution guide](./docs/CONTRIBUTING.md)
 make setup
 ```
 
+## Install foundry
+To be able to verify and compare the abi and bytecode of the test contracts, first make sure you have foundry installed:
```

Comment:
"To compile" ? we don't do any "verification" right?

Reply:
What do you mean by verification? As it is, if you run make build-foundry everything should compile

Reply:
I don't know, you wrote "to verify and compare", and I was also wondering what you meant by this.
We'll foundry to compile AFAIK, not to do any verification

Reply:
Oh I mean manually verify the ABIs and Bycodes that are being used at the test/integration folder